### DISCUSSION TITLE
Mesh start code?

### DISCUSSION THREAD

**Discussion by Horgul123** [mriscoc/Ender3V2S1#1072] [Q&A] [answered]
Normally I create a mesh and save it directly on the printer. I thought the printer would use it then.....
 Now I saw in ja post that I need to add something to the start code ("Change M117 Load Bed Mesh to M420 L0 S1"). Do I need this or another code?

**Comment by mriscoc** [comment, marked as answer]
Read the Wiki, it already has the answer to this and maybe other doubts that you may have.

https://github.com/mriscoc/Ender3V2S1/wiki
